crates/hotshot/hotshot/src/types/handle.rs

```diff
@@ -87,7 +87,7 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES> + 'static, V: Versions>
     }
 
     /// obtains a stream to expose to the user
-    pub fn event_stream(&self) -> impl Stream<Item = Event<TYPES>> {
+    pub fn event_stream(&self) -> impl Stream<Item = Event<TYPES>> + use<TYPES, I, V> {
```

Comment:
This change (and similar changes) are due to this: https://doc.rust-lang.org/edition-guide/rust-2024/rpit-lifetime-capture.html